tests/api_workflow/test_api_workflow_upload_dataset.py

```diff
@@ -20,17 +22,22 @@ def setUp(self) -> None:
         self.create_fake_dataset()
         self.api_workflow_client.tags_api.no_tags = 0
 
-    def create_fake_dataset(self):
+    def create_fake_dataset(self, length_of_filepath: int = -1):
         n_data = self.n_data
         self.dataset = torchvision.datasets.FakeData(size=n_data,
                                                      image_size=(3, 32, 32))
 
         self.folder_path = tempfile.mkdtemp()
         sample_names = [f'img_{i}.jpg' for i in range(n_data)]
-        self.sample_names = sample_names
         for sample_idx in range(n_data):
             data = self.dataset[sample_idx]
-            path = os.path.join(self.folder_path, sample_names[sample_idx])
+            sample_name = sample_names[sample_idx]
+            path = os.path.join(self.folder_path, sample_name)
+
+            if length_of_filepath > len(path):
+                n_missing_chars = length_of_filepath - len(path)
+                path = path.replace('.jpg', 'x' * n_missing_chars + '.jpg')
```

Comment:
I know this is a "fake" thing but just to be sure when replacing things, ensure that it is where you expect it to be. E.g at the end of the filename. (as a file could be called `hello.jpg.png`). So I would suggest replacing it with a regex `/\.jpg$/` just incase someone would copy this code..

Reply:
Good point, I safeguarded it.